languages/aliases.ini

```diff
@@ -0,0 +1,13 @@
+bulk_delete = "delete_selected"
+bulk_delete_myresearch = "bulk_delete"
```

Comment:
Would it be better to alias this directly to delete_selected instead of aliasing an alias? Or do you prefer it this way?

Reply:
As long as we only have a bulk delete button in the favorites we can do this. Done.